core/src/main/java/io/github/fourlastor/game/demo/round/monster/MonsterAbilities.kt

```diff
@@ -76,7 +76,7 @@ interface MonsterAbilities {
         override fun create(): List<Abilities.Description> {
             return listOf(
                 descriptions.move,
-                descriptions.chargeBeam,
+                descriptions.chargeBeam
```

Comment:
keep the last comma, it makes future diffs simpler (as the only changed line is an added/removed line even if it's the last line)